src/wazuh_qa_framework/system/wazuh_handler.py

```diff
@@ -568,33 +569,49 @@ def restart_managers(self, manager_list, parallel=True):
                 self.restart_manager(manager)
         self.logger.info(f'Managers restarted successfully: {manager_list}')
 
-    def stop_agent(self, host):
+    def stop_agent(self, host, type='gracefully'):
         """Stop agent
 
         Args:
             host (str): Hostname
+            type (str): Type of stop, it can be gracefully or ungracefully. Defaults to gracefully
         """
-        self.logger.debug(f'Stopping agent {host}')
-        service_name = WAZUH_ANGENT_WINDOWS_SERVICE_NAME if is_windows(host) else 'wazuh-agent'
+        self.logger.debug(f'Stopping agent {type} {host}')
+        service_name = WAZUH_ANGENT_WINDOWS_SERVICE_NAME if self.is_windows(host) else 'wazuh-agent'
```

Comment:
Previous typo. Please fix it in all the iterations
```suggestion
        service_name = WAZUH_AGENT_WINDOWS_SERVICE_NAME if self.is_windows(host) else 'wazuh-agent'
```

Reply:
Done in https://github.com/wazuh/qa-system-framework/pull/72/commits/ab390ad2c27a3f205a50472e7c0c3a9dc0368f8c